### ISSUE TITLE
release 0.6.5 breaks buttons

### ISSUE THREAD

**Issue by BPMinaker** [JunoLab/language-weave#35]
I think after update to 0.6.5 tool bar buttons are broken, at least for my .jl scripts.  Was working great on 0.6.4, even though the html button was invisible.  0.6.5 brings back the html button, but no functionality for either button.

**Comment by aviatesk** [comment]
now the command is not defined for the julia scope, since you only need to use that for the weave scope.

**Comment by BPMinaker** [comment]
I'm sorry but I'm confused.  If I have a .jl file open in the editor, how
do I issue a weave command in the weave scope? I tried calling 'using
Weave' on the REPL command line, and then using the bottom menu to change
to the Weave scope, but still the buttons don't function. It seems to me
that the most sensible use case would be that I could just open a .jl file,
and then push the weave button. Can I do something similar to retain that
functionality?

I hope I'm not overly critical; I think Weave is awesome, and anticipate
lots of value from using it.

Thanks for your help.

BM

**Comment by pfitzseb** [comment]
Well, `.jl` technically aren't weavable -- you should be in a `.jmd` or similar file (or set the editor's grammar to one of the other `Weave` ones).

That said, I'm not sure if we need to be so strict here -- I don't think there's any real downside to allowing editors with an arbitrary grammar to be weavable.

**Comment by BPMinaker** [comment]
Ok thanks.  Just to be clear, I'm not implying I'll want to weave any
arbitrary .jl file, but my use case is taking something very similar to the
FIR_design.jl example given in the weave package, and making it into a pdf.
Ideally, I could do that with one (or a few?) clicks.

**Comment by aviatesk** [comment]
> That said, I'm not sure if we need to be so strict here

yep, the only reason I made those changes is that not to allow users to dispatch the command on scopes that the commands won't work.

>  but my use case is taking something very similar to the FIR_design.jl example given in the weave package

so is there any case when you want to use the command for .jl files ?
I'm really okay to reregister the command to the workspace, then.

**Comment by BPMinaker** [comment]
Yes, please. If you could re-register, I would be very happy about that.  I
prefer that my scripts maintain the .jl extension, and the ability to run
as regular Julia codes, or be weaved to pdf with the comments included.

Thanks for the super prompt response!

BM
